e2e/specs/audit/auditFlow.spec.ts

```diff
@@ -0,0 +1,165 @@
+                    await audit.attachDocument(testData.getFullPath(`/attachments/${attachName}`));
+                    return expect(audit.isAttachAdded(attachName)).toBe(true, `${attachName} is not added`);
+                });
+
+                it('Can upload attachments', async () => {
+                    await audit.uploadAttachment(attachName);
+                    await expect(audit.notification.isSuccess()).toBe(true, 'Success message is not shown!');
+                    await expect(audit.notification.getContent()).toBe(`'${attachName}' file was uploaded!`, 'Message is wrong!');
+                    await audit.notification.close();
+                    await expect(audit.isAttachUploaded(attachName)).toBe(true, `${attachName} is not uploaded!`);
+                    return expect(audit.isFinishProgressEnabled()).toBe(true, 'Finish progress should be availabe!');
+                });
+            });
+
+            describe(`In Review:`, () => {
+                it('Can finish progress when all data is passed', async () => {
+                    await audit.finishAudit();
+                    await expect(audit.modal.isVisible()).toBe(true, 'No Confirmation modal for Finish Audit action!');
+                    await audit.modal.clickActionBtn('yes');
+                    await expect(audit.getStatus()).toBe(audit.statuses.inReview, 'Audit is not In Review status!');
+                    await expect(audit.notification.isSuccess()).toBe(true, 'Success message is not shown!');
+                    await audit.notification.close();
+                });
+
+                it('Can add changes to in review audit', async () => {
+                    const summary = 'Some New Text';
+                    await audit.setSummary(summary);
+                    await audit.clickSaveSummary();
+                    await audit.notification.close();
+                    await expect(audit.getSummary()).toBe(summary, 'Summary is not updatet');
+                });
+
+                it('Submit button is available for only Audit Admin', async () => {
+                    expect(await audit.isSubmitButtonPresent())
+                        .toBe(user.user_name === auditAdmin.user_name, `Submit button availabilty is wrong for ${user.user_name}`)
+                });
+            });
+
+            describe(`Submitted:`, () => {
+                it('Can submit audit', async () => {
+                    if (user.user_name !== auditAdmin.user_name) {
+                        await projectsList.menuBar.clickLogOut();
+                        await logInPage.logIn(auditAdmin.user_name, auditAdmin.password);
+                        await audit.open(project.name, audit.statuses.inReview);
+                    }
+                    await audit.submitAudit();
+                    await expect(audit.modal.isVisible()).toBe(true, 'No Confirmation modal for Submit Audit action!');
+                    await audit.modal.clickActionBtn('yes');
```

Comment:
I think you can create clickYesButton method to avoid of using string value 'yes'